modules/dxp/apps/saml/saml-web/src/main/resources/META-INF/resources/admin/certificate_info.jsp

```diff
@@ -70,14 +70,27 @@ String introKey = StringPool.BLANK;
 
 		<aui:form action="<%= deleteCertificateURL %>">
 			<aui:button-row>
-				<aui:button onClick='<%= liferayPortletResponse.getNamespace() + "showCertificateDialog('" + replaceCertificateURL + "');" %>' value="replace-certificate" />
+				<aui:button id="" onClick='<%= liferayPortletResponse.getNamespace() + "showCertificateDialog('" + replaceCertificateURL + "');" %>' value="replace-certificate" />
```

Comment:
Is this empty `id` necessary? The button below doesn't have an `id` defined so it doesn't seem to be a required property.

Reply:
Nope, thanks for finding :)